### DISCUSSION TITLE
Components 디렉터리 안에 파일명의 postfix에 대해 어떻게 생각하시나요..?!

### DISCUSSION THREAD

**Discussion by kkojae91** [woowacourse-study/2022-code-review-study#49] [Youtube]
리뷰어님 : [#90](https://github.com/woowacourse/javascript-youtube-classroom/pull/90#discussion_r825309804) 개인적으로는 XXXComponent postfix 는 이미 Component 역할을 하고 있고 파일 네이밍으로도 유추가 가능해서.. 없어지는 게 좋을 거 같습니당

이 부분은 취향 차이가 있을 수 있는것 같지만, 
저는 개인적으로는 VideoContainerComponents, 혹은 VideoContainerView 이렇게 명시적인게 좋다고 생각했는데, 리뷰어님 말을 들어보면 또 굳이 View, Components라는 postfix가 붙어야 하나?? 라고 생각이 드네요!

여러분들의 선택은 어떻게 되나요..?!
```js
// Components/AppComponents
export default class AppComponents {}

// Components/VideoContainerComponents
export default class VideoContainerComponents {}
```

```js
// Components/App.js
export default class App {}

// Components/VideoContainer
export default class VideoContainer {}
```

**Comment by moonheekim0118** [comment]
저는 Postfix를 없애는걸 선호합니다. 
이미 폴더명으로 해당 폴더 내부 파일의 역할 (Component, View)를 명시해주고 있기 때문에요!
그리고 사람이 읽을 때에도 어차피 < Component - 파일명> 이렇게 읽혀서 postfix를 붙일 필요는 없어보여요 ~

**Reply by byhhh2** [reply]
공감합니다 `import`문을 보고도 알 수 있기 때문에 딱히 postfix가 필요없다고 생각돼요.

```js
import App from '../Components/App.js' // 아~ App은 컴포넌트구나
```

**Comment by KangYunHo1221** [comment]
취향차 아닐까요?

전 쓰는 것을 좋아합니다. 대상만 보고 어디 위치에 있을지, 무슨 역할을 할지 유추가 가능하고

같은 appView랑 appModel이 둘다 app을 쓰게 되면 오히려 혼란스러울거 같아요.

**Reply by KangYunHo1221** [reply]
가독성을 해칠정도로 길거나 이상하지 않다면 마르코가 맞는거 같아요 전
